AudioQnA/docker_compose/intel/hpu/gaudi/compose.yaml

```diff
@@ -12,7 +12,8 @@ services:
       no_proxy: ${no_proxy}
       http_proxy: ${http_proxy}
       https_proxy: ${https_proxy}
-      HABANA_VISIBLE_DEVICES: all
+
+      HABANA_VISIBLE_DEVICES: 0
```

Comment:
Why use a fixed card number?  Setting to 'all' allows the system to automatically allocate a card, so I don't think your code change is appropriate.

Reply:
@psurabh please give your comments, this change seems not reasonable for us.